legal-api/src/legal_api/models/furnishing.py

```diff
@@ -125,15 +124,14 @@ def find_by(cls,  # pylint: disable=too-many-arguments
         if status:
             query = query.filter(Furnishing.status == status)
 
-        if grouping_identifier:
-            query = query.filter(Furnishing.grouping_identifier == grouping_identifier)
-
         if furnishing_group_id:
             query = query.filter(Furnishing.furnishing_group_id == furnishing_group_id)
 
         return query.all()
 
     @classmethod
-    def get_next_grouping_identifier(cls):
-        """Return the next grouping_identifier from the sequence."""
-        return db.session.execute("SELECT nextval('grouping_identifier')").scalar()
+    def get_furnishing_group_id(cls):
+        """Return thefurnishing_group_id."""
+        furnishing_group = FurnishingGroup()
+        furnishing_group.save()
+        return furnishing_group.id
```

Comment:
I think there will be more processes like creating the XML_Payload, and others too.
So, I'd like to keep this function for now.